sycl/test-e2e/ESIMD/rdtsc.cpp

```diff
@@ -41,9 +41,9 @@ int test_rdtsc() {
       auto Kernel = ([=](sycl::nd_item<1> ndi) [[intel::sycl_explicit_simd]] {
         using namespace sycl::ext::intel::esimd;
         auto Idx = ndi.get_global_id(0);
-        uint64_t StartCounter = sycl::ext::intel::experimental::esimd::rdtsc();
```

Comment:
nit: maybe we call both experimental and non-experimental based on `Idx % 2` or something